blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md

```diff
@@ -0,0 +1,283 @@
+latest updates and allows you to actively participate in shaping Dasharo's
+journey. It's a small way to say thank you for your trust and support in us. If
+you are interested, you can [become a subscriber
+here](https://shop.3mdeb.com/product-category/dasharo-entry-subscription/).
+
+We are already channeling our energies toward the next release,
+[v1.1.3](https://github.com/Dasharo/dasharo-issues/milestone/12), nurturing it
+with more improvements and refinements.
+
+### Dasharo Community Releases (DCR)
+
+But worry not if a subscription is not for you! We are equally committed to our
+broader community. We have the Dasharo Community Releases lined up,
+built, and published annually, offering a compilation of the features and fixes
+introduced in the Dasharo Entry Subscription Releases. The next in line is the
+v1.2.0 release, a milestone we are eagerly working towards. To stay updated
+with the latest community releases, subscribe to our [free mailing
+list](https://newsletter.3mdeb.com/subscription/aKgTJjYEA).
+
+We are here because of your support and faith in us. Each release, including
+v1.1.2, is a step towards a more secure, user-friendly, and trustworthy
+firmware solution, a journey we are proud to have you be a part of. Thank you
+for being with us on this adventure!
+
+### Key Changes in the New Release
+
+Over the past few months, we have been diligently working to address issues
+reported by the community and introduce new features to enhance the firmware's
+functionality. Here are the critical updates:
+
+## Added
+
+### [MSI FlashBIOS Recovery Support](https://docs.dasharo.com/unified/msi/recovery/#using-msi-flashbios-button)
+
+For coreboot hackers and enthusiasts who are apprehensive about bricking their
+hardware while flashing open-source firmware, the MSI FlashBIOS Recovery
+Support comes as a reassuring feature. As detailed in our documentation, it
+eliminates the risk of a permanent brick, offering a straightforward recovery
+pathway in case of BIOS corruption. This feature, demonstrated in the following
+video, ensures you can experiment and optimize without the looming fear of
+losing your hardware functionality. It's a safety net that encourages
+exploration while maintaining system integrity, a boon for the open-source
+firmware community.
+
+{{< youtube iTkXunUAriE >}}
+
+Of course, that means some microcontroller sitting on the mainboard can read
+the filesystem and flash BIOS SPI under certain circumstances and have access
+to a specific USB port. That microcontroller does not run open-source firmware
+yet and could be leveraged by attackers with the ability to update it or have
+physical access. The only reasonable protection for Intel platforms is Intel
+Boot Guard with owner keys fused into the chipset, which we deliver as part of
+our Dasharo Enterprise offering.
+
+### Raptor Lake-S CPU Support
+
+We've started to add support for the upcoming Raptor Lake-S CPUs in Dasharo.
```

Comment:
```suggestion
We've added support for the Raptor Lake-S CPUs in Dasharo.
```

We finished, not started. Raptor Lake-S was upcoming a year ago (release date was 20th October 2022), now the Raptor Lake-S Refresh is upcoming.